crates/sovereign-sdk/adapters/mock-da/src/service.rs

```diff
@@ -221,7 +221,7 @@ impl MockDaService {
         let blob = MockBlob::new_with_zkp_proof(
             blob.to_vec(),
             zkp_proof,
-            self.sequencer_da_address,
+            self.sequencer_da_address.clone(),
```

Comment:
I don't quite understand why our addresses (mock/regular) turned from [u8; 32] into Vec<u8>.

Reply:
compressed bitcoin da addresses are 33 bytes (first byte for parity), so 32 bytes was incorrect. but when I changed it to [u8; 33] all hell broke loose, hence, temporarily I just changed it to be a Vec, so current 32 byte addresses would work, and I can use 33 bytes wherever I need to.